src/dev/build/tasks/os_packages/docker_generator/resources/bin/opensearch-dashboards-docker

```diff
@@ -150,6 +150,7 @@ opensearch_dashboards_vars=(
     telemetry.optInStatusUrl
     telemetry.sendUsageFrom
     wizard.enabled
+    data_source.enabled
```

Comment:
shall we add all configurations? so that users doesn't need to set those settings via yml config

Reply:
good point